test/util/bspNet/helpers.ts

```diff
@@ -203,7 +203,11 @@ export const addBsp = async (
 
   //Give it some balance.
   const amount = 10000n * 10n ** 12n;
-  await sealBlock(api, api.tx.sudo.sudo(api.tx.balances.forceSetBalance(bspKey.address, amount)));
+  await sealBlock(
+    api,
+    api.tx.sudo.sudo(api.tx.balances.forceSetBalance(bspKey.address, amount)),
+    options?.signer
+  );
```

Comment:
This is a sudo call, could you add a small comment on why now you are passing the signer? (not alice - sudo account)

Reply:
That is better, yes. This function also should always recieve the sudoSigner, right?
we currently have things like:
`hash = await call.signAndSend(signer || alice, {`

Which would could lead devs to dont pass the user to sealBlock in the evm tests, and then trying to sign with substrate alice.

regarding the comment, I was saying a small comment regarding that the user needs to be passed as it can be evm or substrate (something like that)